projects/ad74413r/src/platform/maxim/main.c

```diff
@@ -55,31 +49,16 @@ int main()
 {
 	int ret = -EINVAL;
 
-#ifdef DUMMY_EXAMPLE
+#ifdef EXAMPLE_dummy_example
```

Comment:
this macro is automatically generated and passed to C preprocessor from the user:

`make EXAMPLE=foo` will define a C preprocessor macro `EXAMPLE_foo` (Case sensitive of course) where `foo` is the name of the folder in which the example code resides under `src/examples/foo`